libraries/src/MVC/Controller/BaseController.php

```diff
@@ -1061,4 +1058,30 @@ public function setRedirect($url, $msg = null, $type = null)
 
         return $this;
     }
+
+    /**
+     * Method to set the View Models
+     * 
+     * This function is provided as a default implementation,
+     * and only set one Model in the view (that with the same prefix/sufix than the view).
+     * In case you want to set several Models for your view,
+     * you will need to override it in your DisplayController controller.
+     *
+     * @param   Object  $view  The view Object
+     *
+     * @return  void
+     * 
+     * @since   4.2
```

Comment:
```suggestion
     * @since   __DEPLOY_VERSION__
```

`__DEPLOY_VERSION__` will be replaced on build with the correct Joomla version.